sampleapp/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion rootProject.ext.targetSdkVersion
+
+    defaultConfig {
+        applicationId "com.zendesk.sampleapp"
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "1.0"
+
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+}
+
+repositories {
+    google()
+    jcenter()
+    mavenCentral()
```

Comment:
I'm pretty sure you can remove `mavenCentral()` if you have `jcenter`. jcenter is a superset of mavenCentral, so I think all this would do is go search through mavenCentral if it can't find something in jcenter, which obviously would be wasted work in that case. So it might just slow you down. Try removing it maybe?